renaming qgis variables to mm_ (#607)
MerginMaps/docs: src/layer/plugin-variables/index.md

The <QGISPluginName /> adds several variables that can be used in QGIS expressions:

| Variable name               | Sample value                  | Scope   | Description |
|-----------------------------|-------------------------------|---------|-------------

> | `@mm_full_name`          | `martin`                      | global  | Full name of the user currently logged in to <MainPlatformNameLink /> |

Reviewer: ```suggestion
| `@mm_full_name`          | `martin second name`                      | global  | Full name of the currently logged in user, blank if the full name is not set |
```

| `@mm_username`          | `martin`                      | global  | Username of the user currently logged in to <MainPlatformNameLink /> |
| `@mm_user_email`        | `martin@example.com` | global  | Email of the user currently logged in to <MainPlatformName />|
| `@mm_url`               | `https://app.merginmaps.com`  | global  | URL of the <MainPlatformName /> service |
| `@mm_project_name`      | `Tree survey`                 | project | Name of the active <MainPlatformName /> project  |

> | `@mm_project_owner`     | `martin`                      | project | Name of the owner of the active project |

Reviewer: Please remove this line, this is some old behaviour which is not supported anymore

```suggestion
```

> | `@mm_project_full_name` | `martin/Tree survey`          | project | Owner and project name joined with a forward slash |

Reviewer: ```suggestion
| `@mm_project_full_name` | `martin/Tree survey`          | project | Workspace and project name joined with a forward slash |
```
